### DISCUSSION TITLE
Need to confirm that grizzly does NOT work for alerts....

### DISCUSSION THREAD

**Discussion by kingDaddyFreshness** [grafana/grizzly#232] [General]
Hello all!

I'm new to grizzly.  I'm looking at grizzly as a tool that I can use to download my dashboards from a dev environment and then push them to a prod environments.  I have my own scripts that do this but want to get out of the business of maintaining my own scripts and grizzly seems like it can do that for me.  I've played with it a bit and successfully downloaded datasources, dashboards, and folders.  But I also need to download (and upload) alerts.

As far as I can tell there are a few types of alerts in grafana (version 8+).  These include "legacy" alerts that are associated with a dashboard and panel; "new" alerts that are associated with a folder; loki alerts (which I don't use) and Prometheus alerts (which I don't use).  I have not been able to successfully download either the "legacy" or the "new" type of alerts.  Can someone verify that grizzly does/doesn't support this?  If it does support this, could someone provide a command line example?

Thanks!

-john

**Comment by malcolmholmes** [comment]
Grizzly supports prometheus alerts when directed at either Mimir or Cortex ruler. It does not yet support the newer "Unified Alerts" in Grafana. It is a clear wish, but it'll likely take us a while before we get there.

**Reply by defanator** [reply]
@malcolmholmes can you share an example on how to pull all existing alerts via Mimir endpoint hosted at Grafana Cloud? I've been having hard times trying to do that using amtool and mimirtool recently, hope that grr could do the thing. TIA!
